test_suite/cloud/test_aws.py

```diff
@@ -301,6 +301,10 @@ def test_required_packages_are_installed(self, host):
     @pytest.mark.pub
     @pytest.mark.run_on(['rhel'])
     def test_rhui_pkg_is_installed(self, host):
+        with host.sudo():
+            # Workaround for "yum and rpm get a db5 error"
+            assert host.run('rm -f /var/lib/rpm/__db.*').succeeded, 'failed to remove the RPM db files'
+            assert host.run('rpm --rebuilddb').succeeded, 'failed to rebuild the RPM db files'
```

Comment:
You can do this instead, to directly check that command did not fail:
```suggestion
            assert host.run_test('rpm --rebuilddb'), 'Failed to rebuild the RPM db files'
```
(same for the line above)
More info here: https://testinfra.readthedocs.io/en/latest/modules.html#testinfra.host.Host.run_test

Reply:
ACK